Qualification/Input/Content/Moclobemide-Omeprazole-DDI.md

```diff
@@ -1,9 +1,9 @@
-The moclobemide-omeprazole interaction was evaluated using clinical DDI studies listed in [Table 5](#Table 5).
+The moclobemide-omeprazole interaction was evaluated using clinical DDI studies listed in [Table 5](#table-5).
```

Comment:
Let's keep it like this for now.
But for the future: I don't think it makes sense to insert hyperlink to the table which is directly below.
And: because here the table caption is **below** the table: when clicking on the link, the table is out of sight and the user must scroll up to see it. E.g. when I click on the link table-5 above, that's what I see:
![grafik](https://user-images.githubusercontent.com/25061876/167262504-a71b334a-9724-4c58-9013-c26c5d9b8349.png)

By the way: R-RE will create all table captions **above** the table (and figure captions **below** the figure).
This seems to be the common practice in scientific papers. Thus our "static" tables and figures should follow the same concept. I will create issues for that.

<details><summary><b>Example guidance for captions from https://gradstudents.carleton.ca/2014/guidelines-using-figures-tables-scientific-engineering-thesis <b></summary>

![grafik](https://user-images.githubusercontent.com/25061876/167262719-84f9ab8a-0073-49ae-8b6b-89c3045529e5.png)
</details>